cmd/state-tray/main.go

```diff
@@ -57,28 +72,56 @@ func run() error {
 
 	systray.AddSeparator()
 
+	mProjects := systray.AddMenuItem(locale.Tl("tray_projects_title", "Local Projects"), "")
+	mReload := mProjects.AddSubMenuItem("Reload", "Reload the local projects listing")
+	localProjectsUpdater := menu.NewLocalProjectsUpdater(mProjects)
+
+	localProjects, err := model.LocalProjects()
+	if err != nil {
+		logging.Error("Could not get local projects listing: %v", err)
+	}
+	localProjectsUpdater.Update(localProjects)
+
+	systray.AddSeparator()
+
 	mQuit := systray.AddMenuItem(locale.Tl("tray_exit", "Exit"), "")
 
 	for {
 		select {
 		case <-mAbout.ClickedCh:
 			logging.Debug("About event")
-			err := open.Prompt("state --version")
+			err = open.Prompt("state --version")
 			if err != nil {
-				logging.Error("Could not open command prompt, got error: %v", err)
+				logging.Error("Could not open command prompt: %v", err)
 			}
 		case <-mDoc.ClickedCh:
 			logging.Debug("Documentation event")
-			// Not implemented
+			err = open.Browser("https://docs.activestate.com/platform/state/")
+			if err != nil {
+				logging.Error("Could not open documentation url: %v", err)
+			}
 		case <-mLearn.ClickedCh:
 			logging.Debug("Learn event")
-			// Not implemented
+			err = open.Browser("https://www.activestate.com/blog/")
+			if err != nil {
+				logging.Error("Could not open blog url: %v", err)
+			}
 		case <-mSupport.ClickedCh:
 			logging.Debug("Support event")
-			// Not implemented
+			err = open.Browser("https://www.activestate.com/support/")
+			if err != nil {
+				logging.Error("Could not open support url: %v", err)
+			}
 		case <-mAccount.ClickedCh:
 			logging.Debug("Account event")
 			// Not implemented
+		case <-mReload.ClickedCh:
+			logging.Debug("Projects event")
+			localProjects, err = model.LocalProjects()
+			if err != nil {
+				logging.Error("Could not get local projects listing: %v", err)
```

Comment:
Should return here, else line 124 will likely lead to a panic.

Reply:
The response on error will be `nil` and line 124 will not panic. I've tested it with a `nil` response and when it's running without the service, it works in both cases.